client/containers/EventsPanelContainer/index.jsx

```diff
@@ -29,9 +29,10 @@ EventsPanel.propTypes = {
 
 const mapStateToProps = (state) => ({
     // selectors.getShowEventDetails is either true, an event id, or null
-    eventToDetail: selectors.getShowEventDetails(state) === true
-        ? DEFAULT_FORM_VALUES
-        : selectors.getEventToBeDetailed(state),
+    eventToDetail: selectors.getShowEventForm(state) ? selectors.getShowEventDetails(state) === true
```

Comment:
what means "EventForm" in `getShowEventForm` ? Is it this form in the deletion modal ?
This logic is a bit unclear to me, maybe because there are two ternary operations in the same instruction :)